packages/typedoc-plugins/tests/module-fixer/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+const path = require( 'path' );
+const { expect } = require( 'chai' );
+const glob = require( 'fast-glob' );
+const TypeDoc = require( 'typedoc' );
+
+const paths = require( '../utils' );
+
+const FIXTURES_PATH = path.join( paths.ROOT_TEST_DIRECTORY, 'module-fixer', 'fixtures' );
+
+describe( 'module-fixer', function() {
+	this.timeout( 10 * 1000 );
+
+	let conversionResult;
+
+	before( async () => {
+		const sourceFilePatterns = [
+			path.join( FIXTURES_PATH, '**', '*.ts' )
```

Comment:
```suggestion
			path.join( FIXTURES_PATH, '**', '*.ts' ).replace( /\\/g, '/' )
```

Otherwise it will fail on Windows.